Infrastructure/Kubernetes/02-kubernetes-architecture.md

```diff
@@ -0,0 +1,14 @@
+# Intoduction
+
+[Kubernetes Nodes](https://kubernetes.io/docs/concepts/architecture/nodes/)
+[Master Node Communication](https://kubernetes.io/docs/concepts/architecture/master-node-communicatio/)
```

Comment:
> https://kubernetes.io/docs/concepts/architecture/master-node-communicatio

should be `https://kubernetes.io/docs/concepts/architecture/master-node-communication`